apps/full-site-editing/full-site-editing-plugin/full-site-editing/editor/switch-to-draft-removal/index.js

```diff
@@ -0,0 +1,33 @@
+/* global fullSiteEditing */
+
+/**
+ * External dependencies
+ */
+import domReady from '@wordpress/dom-ready';
+
+/**
+ * Removes switch to draft button on templates.
+ *
+ * It doesn't make sense to have this shown on templates since we always want to preserve templates
+ * as published. Drafting or trashing them could break FSE functionality.
+ */
+function removeSwitchToDraftButton() {
+	// Only remove Switch to Draft button on template post type.
+	if ( 'wp_template' !== fullSiteEditing.editorPostType ) {
+		return;
+	}
+
+	const switchToDraftRemoveInterval = setInterval( () => {
+		const draftButton = document.querySelector( '.editor-post-switch-to-draft' );
+
+		if ( ! draftButton ) {
+			return;
+		}
+
+		clearInterval( switchToDraftRemoveInterval );
+
+		draftButton.parentNode.removeChild( draftButton );
```

Comment:
This is causing a `DOMException` when the editor re-renders after a save.

![Screen Shot 2019-08-09 at 11 37 38](https://user-images.githubusercontent.com/233601/62786988-203b5500-ba9a-11e9-91dd-f57d336a8eb7.png)

I've fixed this on #35278 by using `style.dispay = 'none'` instead, because neither this, nor `ReactDOM.unmountComponentAtNode` worked (not a top level node).

Reply:
Thanks, I was thinking whether to go with `style` but this seemed like a bit "cleaner" hack at the time. I'll switch to that instead. 👍

Reply:
Would recommend a CSS hide + overriding the redux action

Reply:
It seems that CSS hide was sufficient to address it completely. Do you still think that we need to override the action?

Reply:
probably okay, but let's document it :D just in case